app/views/decidim/anonymous_codes/admin/code_groups/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="card">
+  <div class="card-divider">
+    <h2 class="card-title">
+      Access code groups
+      <a class="button tiny button--title new-access" href="<%= new_code_group_path %>"><%= t("decidim.anonymous_codes.admin.code_groups.index.new_access_code_group_button") %></a>
+    </h2>
+  </div>
+  <div class="card-section">
+    <% if groups.any? %>
+      <div class="table-scroll">
+        <table class="table-list">
+          <thead>
+            <tr>
+              <th><%= t("decidim.anonymous_codes.admin.code_groups.index.title") %></th>
+              <th><%= t("decidim.anonymous_codes.admin.code_groups.index.expires_at") %></th>
+              <th><%= t("decidim.anonymous_codes.admin.code_groups.index.active") %></th>
+              <th><%= t("decidim.anonymous_codes.admin.code_groups.index.num_of_tockens") %></th>
+              <th><%= t("decidim.anonymous_codes.admin.code_groups.index.max_reuses") %></th>
+              <th></th>
+            </tr>
+          </thead>
+          <tbody>
+              <% groups.each do |group| %>
+            <tr>
+              <td><%= translated_attribute(group.title) %></td>
+              <td><%= l(group.expires_at, format: :decidim_short) %></td>
+              <td><%= t("booleans.#{group.active}") %></td>
+              <td><%= group.tokens.count %></td>
```

Comment:
After mergin #12 we will change this to `group.usage_count`

Reply:
Oh, ok.